app-frontend/src/app/services/scenes/cmrRepository.service.js

```diff
@@ -91,9 +92,21 @@ export default (app) => {
         }
 
         getThumbnail(scene) {
-            return this.$q(resolve => {
+            return this.$q((resolve, reject) => {
                 const thumbnails = this.getThumbnails(scene);
-                resolve(thumbnails.length ? thumbnails[0].href : false);
+                if (thumbnails.length) {
+                    resolve(thumbnails[0].href);
+                } else if (scene.sceneType === 'COG') {
```

Comment:
I think this branch might be fruitless -- while browsing in the CMR repository, the scenes that
come back are necessarily not yet processed, so I don't think they can have `sceneType == 'COG'`.